### DISCUSSION TITLE
Two noob questions (and ReplayGain support via graphic interface)

### DISCUSSION THREAD

**Discussion by MaschineBrennt** [rern/rAudio#1048] [General]
Hi to all.

Sorry guys, I'm new to my raspberry pi 4b, Dac hat and audio players.
I'd like to kindly ask you just two quick questions:

1) I have dozens and dozens of Flac files of music from different sources (albums/cds/digital stores) that have Replaygain "track" tag/metadata (created by foobar2000 using the EBU R128 loudness normalisation algorithm): does rAudio-1 support the activation and recognition of such tags via the Gui **WITHOUT** having to manually edit some MPD configuration text files?
E.g.: Volumio does not allow it (via Gui), you have to manually edit some MPD config files.

2) In your opinion which Dac is better (in terms of audio quality and rAudio-1 support) between HifiBerry Dac2 HD and Allo Boss2?

Many thanks.

**Comment by rern** [comment]
> 1. I have dozens and dozens of Flac files of music from different sources (albums/cds/digital stores) that have Replaygain "track" tag/metadata (created by foobar2000 using the EBU R128 loudness normalisation algorithm): does rAudio-1 support the activation and recognition of such tags via the Gui **WITHOUT** having to manually edit some MPD configuration text files?

Settings > Player > Volume > Replay Gain
